### PR TITLE
ETC changes - new translations / corrections

### PR DESCRIPTION
**Pull request by stvkno**
Translated some lines as well as fixed some errors

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -2884,14 +2884,14 @@ ETC_20150317_002883	Date
 ETC_20150317_002884	Click to go to the item page.
 ETC_20150317_002885	Recent
 ETC_20150317_002886	Help
-ETC_20150317_002887	Enter the team name of your friend
+ETC_20150317_002887	Enter your friend's team name.
```
Comment (Sunaries) [inline ETC.tsv]: Original has no period. What does it look like, e.g. a screenshot?

File: ETC.tsv [hunk 2]
```diff
@@ -245,8 +245,8 @@ ETC_20150317_000244	$Black Drake
 ETC_20150317_000245	$This species was once fairly rare; however, in recent times, their numbers have greatly increased.
 ETC_20150317_000246	$Big Cockatrice
 ETC_20150317_000247	$The Big Cockatrice is quite dangerous, as its glare and breath can petrify any opponent. Engaging this creature should not be taken lightly.
-ETC_20150317_000248	$Green Cockat
-ETC_20150317_000249	$Red Cockat
+ETC_20150317_000248	$Green Cockatoo
```
Comment (Sunaries) [inline ETC.tsv]: Should remain 'Cockat'

File: ETC.tsv [hunk 3]
```diff
@@ -245,8 +245,8 @@ ETC_20150317_000244	$Black Drake
...
+ETC_20150317_000249	$Red Cockatoo
```
Comment (Sunaries) [inline ETC.tsv]: Should remain 'Cockat'

File: ETC.tsv [hunk 4]
```diff
@@ -3304,7 +3304,7 @@ ETC_20150317_003303	Defeat Rexipher and the monsters!
 ETC_20150317_003304	] should be completed to pass through
 ETC_20150317_003305	The song seems to touch the heart so let's copy it
 ETC_20150317_003306	Found the lighted fountain
-ETC_20150317_003307	Putting up
+ETC_20150317_003307	Building..
```
Comment (Sunaries) [inline ETC.tsv]: Remove the periods.

File: ETC.tsv [hunk 5]
```diff
@@ -14203,7 +14203,7 @@ ETC_20150807_014204	{memo X}
 ETC_20150807_014205	{memo X}
 ETC_20150807_014206	Group Settings {img white_right_arrow 18 18}
 ETC_20150807_014207	{@st41b}Fold {img red_up_arrow 16 16} 
-ETC_20150918_014208	Cockat
+ETC_20150918_014208	Cockatoo
```
Comment (Sunaries) [inline ETC.tsv]: Should remain 'Cockat'

File: ETC.tsv [hunk 6]
```diff
@@ -16366,15 +16365,15 @@ ETC_20151022_016366	You can't touch them since they are being protected with the
 ETC_20151022_016367	Using the determination of the guiding owl
 ETC_20151022_016368	Spreading the energies of the forest
 ETC_20151022_016369	You've found the Shaman Doll of the gravekeeper!
-ETC_20151022_016370	You've found the basket of the hidden beef jerkies!{nl}Pour the polluted red water and pollute the beef jerkies!
-ETC_20151022_016371	Polluting the beef jerkies
+ETC_20151022_016370	You've found the hidden basket of beef jerky!{nl}Use the polluted red water to contaminate the beef jerky!
+ETC_20151022_016371	Contaminating the beef jerky..
```
Comment (Sunaries) [inline ETC.tsv]: Remove the periods.

File: ITEM.tsv [hunk 7]
```diff
@@ -7471,13 +7471,13 @@ ITEM_20151022_007470	Rank 6 Card
...
 ITEM_20151022_007473	It seems that you are able to obtain something that is unknown. Right-click to use.
-ITEM_20151022_007474	Disgusting Deathweaver Cube
+ITEM_20151022_007474	Obscene Deathweaver Cube
 ITEM_20151022_007475	Bleak Chapparition Cube
 ITEM_20151022_007476	Earth Temple Shooter Cube
 ITEM_20151022_007477	Earth Canceril Cube
 ITEM_20151022_007478	Earth Archon Cube
 ITEM_20151022_007479	Violent Cerberus Cube
-ITEM_20151022_007480	Noisy Mineloader Cube
+ITEM_20151022_007480	Raw Mineloader Cube
```
Comment (Sunaries) [inline ITEM.tsv]: Can you elaborate the change? The original means 'noisy' or 'loud.

File: ITEM.tsv [hunk 8]
```diff
@@ -8446,29 +8446,29 @@ ITEM_20151102_008445
...
+ITEM_20151102_008463	Experiment Journal: Chapter 1
+ITEM_20151102_008464	The experiment journal created by the wizard in question
+ITEM_20151102_008465	Experiment Journal: Chapter 2
+ITEM_20151102_008466	Experiment Journal: Chapter 3
 ITEM_20151102_008467
 ITEM_20151102_008468
 ITEM_20151102_008469
 ITEM_20151102_008470
-ITEM_20151102_008471
+ITEM_20151102_008471	Statue of Withus
```
Comment (Sunaries) [inline ITEM.tsv]: Withus -> Widas

**Comment by stvkno** [comment]
@Sunaries requested changes have been made

**Comment by Sunaries** [comment]
:+1:

**Comment by imcgames** [comment]
:+1:

**Pull request merged**